.github/workflows/README.md

```diff
@@ -0,0 +1,46 @@
+#### Our Workflows
+
+Additional information on our GitHub Action Workflows can be found in our [Security Policy](https://github.com/nasa/cFS/security/policy).
+
+##### Format Check
+[![Format Check](https://github.com/nasa/osal/actions/workflows/format-check.yml/badge.svg)](https://github.com/nasa/osal/actions/workflows/format-check.yml)
+
+This action uses clang-format-10 to check for format errors.
+
+##### CodeQL Analysis
+[![CodeQL Analaysis](https://github.com/nasa/cfs/actions/workflows/codeql-build.yml/badge.svg)](https://github.com/nasa/cfs/actions/workflows/codeql-build.yml)
+
+This action runs GitHub's industry-leading static analysis engine, CodeQL, against our repository's source code to find security vulnerabilities. It then automatically uploads the results to GitHub so they can be displayed in the repository's code scanning alerts found under the security tab. CodeQL runs an extensible set of [queries](https://github.com/github/codeql), which have been developed by the community and the [GitHub Security Lab](https://securitylab.github.com/) to find common vulnerabilities in your code.
+
+Our CodeQL action uses a configuration file to use specific queries, which can be found at [.github/codeql](https://github.com/nasa/cFS/tree/main/.github/codeql).
+
+
+##### Static Analaysis
+[![Static Analaysis](https://github.com/nasa/cfs/actions/workflows/static-analysis.yml/badge.svg)](https://github.com/nasa/cfs/actions/workflows/static-analysis.yml)
+
+This action runs a static analysis tool for C/C++ code known as cppcheck. Cppcheck is designed to be able to analyze C/C++ code even if it has non-standard syntax, which is common in embedded projects.
+
+##### Deprecated Build, Test, and Run
+[![Deprecated Build, Test, and Run](https://github.com/nasa/cfs/actions/workflows/build-cfs-deprecated.yml/badge.svg)](https://github.com/nasa/cfs/actions/workflows/build-cfs-deprecated.yml)
+
+This action builds, tests, and runs the cFS bundle using deprecated code.
+
+##### Build, Test, and Run [OMIT_DEPRECATED=true]
+[![Build, Test, and Run %5B OMIT_DEPRECATED=true %5B](https://github.com/nasa/cfs/actions/workflows/build-cfs.yml/badge.svg)](https://github.com/nasa/cfs/actions/workflows/build-cfs.yml)
+
+This action builds, tests, and runs the cFS bundle omitting deprecated code.
+
+##### Documentation and Guides
+[![Documentation and Guides](https://github.com/nasa/cfs/actions/workflows/build-documentation.yml/badge.svg)](https://github.com/nasa/cfs/actions/workflows/build-documentation.yml)
+
+This action creates doxygen documents for cFE, cFS users guide, and osal guide.
```

Comment:
maybe add that it also pushes the pdf for pushes to `main`

Reply:
Added: 
> Documentation and Guides is ran for every push and every pull request on all branches of cFS in Github Actions. The workflow pushes the PDFs for every push on the main branch.

Reply:
good, we might also want to add the link to where the documentation is pushed to: 
https://github.com/nasa/cfs/tree/gh-pages